### DISCUSSION TITLE
"Clear ALL data" Explanation

### DISCUSSION THREAD

**Discussion by dalogue1** [Nachtzuster/BirdNET-Pi#155] [General]
I just set up a BirdNET-Pi for a friend and want to initialize the bird database.  Does "Clear ALL data" just remove bird detections or does it reset all the settings as well?

(On a side note is there any documentation for the many options in the application.)

![image](https://github.com/user-attachments/assets/ed879c44-371f-4827-b491-bd33339f85a2)

**Comment by alexbelgium** [comment]
Hi, it executes : https://github.com/Nachtzuster/BirdNET-Pi/blob/main/scripts/clear_all_data.sh

**Reply by dalogue1** [reply]
Thank you
